packages/features/ee/api-keys/components/ApiKeyListItem.tsx

```diff
@@ -43,16 +43,8 @@ const ApiKeyListItem = ({
       <div>
         <p className="font-medium"> {apiKey?.note ? apiKey.note : t("api_key_no_note")}</p>
         <div className="flex items-center space-x-3.5">
-          {!neverExpires && isExpired && (
-            <Badge className="-p-2" variant="red">
-              {t("expired")}
-            </Badge>
-          )}
-          {!isExpired && (
-            <Badge className="-p-2" variant="green">
-              {t("active")}
-            </Badge>
-          )}
+          {!neverExpires && isExpired && <Badge variant="red">{t("expired")}</Badge>}
+          {!isExpired && <Badge variant="green">{t("active")}</Badge>}
```

Comment:
Testing the API Keys section of the settings to show the "needs license" warning, couldn't help to notice the badges were wrongly padded, this is the fix.